src/Admin/AbstractAdmin.php

```diff
@@ -925,6 +928,8 @@ final public function attachAdminClass(FieldDescriptionInterface $fieldDescripti
     }
 
     /**
+     * @param string|int|null $id
```

Comment:
should we allow object here? for uuids.

Reply:
The modelManager `find` does not supports uuids.

Reply:
I can, but since you're using UUID with Sonata, It would be more interesting if you do it with explanation about what need to be changed ?